test/e2e/framework/volume_util.go

```diff
@@ -460,31 +480,40 @@ func TestVolumeClient(client clientset.Interface, config VolumeTestConfig, fsGro
 	}
 	clientPod, err := podsNamespacer.Create(clientPod)
 	if err != nil {
+		By(fmt.Sprint("failed to creat pod ", err))
```

Comment:
Debugging messages should use Logf() instead of By()

Reply:
fixed